Dmitriy_Vysotskyy2.0/Hooks/BaseHook.cs

```diff
@@ -0,0 +1,27 @@
+using Dmitriy_Vysotskyy2._0.Common;
+using Dmitriy_Vysotskyy2._0.PageObjects;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Edge;
+using TechTalk.SpecFlow;
+
+namespace Dmitriy_Vysotskyy2._0.Hooks;
+
+[Binding]
+public sealed class BaseHook : FeatureHelper
+{
+    [BeforeScenario]
+    public void BeforeScenario()
+    {
+        _driver = new ChromeDriver();
```

Comment:
As I mentioned in the task: you should implement Singleton pattern + you should have a possibility to run both Chrome/Edge drivers. 

Here you can find example of Singleton pattern -> https://refactoring.guru/uk/design-patterns/singleton/csharp/example

Let's also add class called BrowserConfig, where we will have 1 property called BrowserType. 
![image](https://github.com/dimavys/Dmitriy_Vysotskyy2.0/assets/47978115/032aee8d-06a3-440e-9f8e-3e3d515fde0b)

BrowserType is enum, you should create it by yourself. Depening on value in this config we will create either Chrome / Edge driver.

Reply:
You migh face next issue: 
![image](https://github.com/dimavys/Dmitriy_Vysotskyy2.0/assets/47978115/b4bd92d0-a0b4-49cd-9943-87d717811b8d)

After driver is quit it is still not null. For this you can add some property like this 
![image](https://github.com/dimavys/Dmitriy_Vysotskyy2.0/assets/47978115/5e1c76a9-04dd-4070-af4f-a368626f485e)

Reply:
Also please add simple factory method here. Depending on BrowserType we will return either chrome / edge driver with some propties like timeour, windows maximize, etc